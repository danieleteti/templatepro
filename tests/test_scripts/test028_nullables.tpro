List of objects with nullables
{{for obj in objects_nullables}}
{{:@@index}}: [{{:obj.NullString}}] [{{:obj.NullBoolean}}] [{{:obj.NullInt32}}] [{{:obj.NullDate}}]
---
{{endfor}}
