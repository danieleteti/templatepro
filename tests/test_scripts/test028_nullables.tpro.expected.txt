List of objects with nullables
: [true] [False] [1] [2011-11-17]
---
: [] [] [] []
---
